src/styles/main.scss

```
@@ -0,0 +1,63 @@
+$size-watch: 300px;
+$size-center: 15px;
```

Pay attantion to variables naming
```suggestion
$centered-point-size: 15px;
```